eva_cttv_pipeline/clinvar_xml_utils/clinvar_xml_utils/__init__.py

```diff
@@ -0,0 +1 @@
+from .clinvar_xml_utils import *
```

Comment:
This is some namespace stitching that avoids one repetition of `clinvar_xml_utils` in the imports, though I admit it's mostly aesthetic... an alternative would be to think of some other name for the `clinvar_xml_utils.py` file but I can't think of anything right now.

Reply:
I don't actually like the term utils (too generic). The package is mainly a reader so we could it something like `clinvar_xml_io.reader`.

Reply:
Oh I see, you can shorten the other imports by adding it to its module's ``__init__.py``. Neat-o

Reply:
@tcezard I agree that `utils` isn't a great name and `io` probably makes more sense (it actually would be quite handy for us to be able to get the "o" part working, right now I have a couple hacky scripts I use to generate test data and other subsets for analysis).

Not sure about renaming the file `reader`, I was thinking of actually just splitting that file into its various classes (record, measure, trait...) as it's quite long and hard to navigate right now.

Reply:
I think at this point we only need to agree on a package name before we start pushing to pypi.
If we want to refactor the classes we should do it subsequently.